### ISSUE TITLE
distort points

### ISSUE THREAD

**Issue by fanhuigui** [hku-mars/STD#17]
Hello, I would like to ask if this point cloud data has already been distorted before processing

**Comment by ChongjianYUAN** [comment]
> Hello, I would like to ask if this point cloud data has already been distorted before processing

yes. To run our your private dataset, you should provide undistorted point cloud.

**Comment by fanhuigui** [comment]
> > Hello, I would like to ask if this point cloud data has already been distorted before processing
> 
> yes. To run our your private dataset, you should provide undistorted point cloud.

Hello, I applied your loopback detection module to FAST-LIO2 and kept making mistakes. It took me a long time to fix it, and I don't know where the problem lies
![2023-05-29 14-26-03 的屏幕截图](https://github.com/hku-mars/STD/assets/100503381/0eac4db3-abd5-4882-9732-e54c47181a6b)

>
